Add suboptimal 2D convolution

meshtag opened the pull request:
This pull request intends to add a suboptimal implementation of generic 2D convolution. This is done for the purpose of giving a rough idea to Fys about how to work with python arrays/loops, etc. Fys will be asked to improve this implementation and complete tasks related to convolution on top of it.

gautam-dev-maker commented on cv-basics/Suboptimal_Convolution.ipynb:
```diff
@@ -0,0 +1,90 @@
+{
```
<p>Add the theory of supoptimal convolution ,some visual representation of Maths behind it.</p><p> </p><p>Also add the Heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

meshtag replied:
@gautam-dev-maker , theory and math related to convolution is covered nicely in convolution PPT. This code will also make its place there. My purpose of creating this PR was : 
 - To collect reviews/opinions about the implementation
 - Group this small part with rest of the code facilating ease of accessibility for future use and maintainance

SAtacker replied:
@meshtag  What @gautam-dev-maker means is that we should have added everything in the .ipynb itself instead of a PPT which will not be visible to outsiders

SAtacker commented:
Kindly view #23

SAtacker reviewed (commented):
It still does not contain all the theory hence view #23

meshtag commented:
@SAtacker , shall we keep this file at the end of Jupyter notebook provided in #23 ?

SAtacker commented:
What file are you talking about?
I have renamed yours into convolution and added all the theory part

meshtag commented:
> I have renamed yours into convolution and added all the theory part

Alright, I realised that just now. Apologies for not realising it earlier. I am closing this PR for now since its content has been added in #23.
